src/utils/formatters.ts

```diff
@@ -11,6 +11,9 @@ export function formatNumber(n: number) {
     case -Infinity:
       return '-'
     default:
+      if (Number.isNaN(n)) {
```

Comment:
Is it likely that someone downloads a 0 population statistic though? Should we have a "0 population in service area" instead? Does that even happen?

Reply:
This formatter is a last line of defense against errors (maybe there're a frontend bug and the number was computed incorrectly, or network is down, or backend bug, or missing error in DB). We should guard against those failure modes individually too, this is just so if something does fail, the UI doesn't break.